src/core/Button/Button.baseStyles.tsx

```diff
@@ -139,6 +140,9 @@ export const baseStyles = withSuomifiTheme(
       margin-left: ${theme.spacing.insetM};
     }
   }
+  &.fi-button--disabled > .fi-button_icon {
```

Comment:
Could this be nested under disabled styles in starting in line 108?

Reply:
It was first there but then it complained about the descendant style rule.